README.md

```diff
@@ -10,6 +10,16 @@ This plugin once installed will be added to the mu-plugin folder so it is ready
 ### Creating new blocks
 In order to create new blocks, you can add a new class to your theme or plugin and extend the `CreodeBlocks\Block` class.
 
+If you are utilising WP_CLI for your project, you can use the following command to create a new block class:
+
+```bash
+wp make-block "<block-name>"
+```
```

Comment:
Should there be an example of the theme argument? and a real example showing the required formatting of block-name, eg, spaces or no spaces, capitalised listed or not?